modules/persistence/src/services/pages/index.ts

```diff
@@ -267,9 +267,16 @@ const updatePages = async (pages: Page[], collection: string, githubUser: string
   console.time(timerLabel);
 
   try {
+    // TEMPORARY FIX FOR NETLIFY BUILDS
+    // TODO: DOP-5405 remove parser user from page id
+    const pageIdArr = pages[0].page_id.split('/').slice(0, 3);
```

Comment:
NIT: It'll probably be better to use `replace` as the index of where `buildbot` might be could change.